vault/resource_identity_entity_alias.go

```diff
@@ -75,32 +76,22 @@ func identityEntityAliasCreate(ctx context.Context, d *schema.ResourceData, meta
 
 	diags := diag.Diagnostics{}
 
-	var duplicates []string
-
 	mountAccessor := data["mount_accessor"].(string)
-	aliases, err := entity.FindAliases(client, &entity.FindAliasParams{
```

Comment:
This is the only other use of this function is [vault/resource_identity_entity_alias_test.go#L191](https://github.com/hashicorp/terraform-provider-vault/blob/main/vault/resource_identity_entity_alias_test.go#L191). I'm not sure if it is better to just replace that reference with a more specific helper function or just leave it as is.

The current implementation of `entity.FindAliases` is only really dangerous when running against a server with a large number of entities, which won't be the case when running the tests.

Reply:
Good point, we will take a closer look after your PR is merged!